main.go

```diff
@@ -68,13 +68,11 @@ func main() {
 	}
 
 	if flags.dry == false {
-		resolve := output.Stdout("Attempting to push new tag to GitHub...")
+		output.Stdout("Attempting to push new tag to GitHub...")
```

Comment:
It seems like this is still going to output this line. So I don't really see how this fixes the issue?